### DISCUSSION TITLE
Happy Halloween to all China Lantern's users with Chinese Vampire! 万圣节快乐!

### DISCUSSION THREAD

**Discussion by MaxMKMax** [getlantern/lantern-forum-cn#1010] [General]
![Happy Halloween Lantern](https://github.com/user-attachments/assets/21d9fb87-5b2f-4744-8e76-dc432d4e5927)

**Comment by guofulibai123** [comment]


**Reply by MaxMKMax** [reply]
@guofulibai123 您好. 我不是负责退款的. 是美国团队那一边. 我能控制进度的能力有限. 
那您提供支付单据和账号邮箱给我. 我帮您再一次提交问题上去.
